docs/input/docs/reference/build-servers/bitbucket-pipelines.md

```diff
@@ -0,0 +1,78 @@
+---
+Order: 35
+Title: BitBucket Pipelines
+Description: Details on the Atlassian BitBucket Pipelines support in GitVersion
+RedirectFrom: docs/build-server-support/build-server/bitbucket-pipelines
```

Comment:
Since this is a brand new page, this RedirectFrom is not required. The RedirectFrom on the existing pages are needed since the pages have moved around a fair bit since the docs were first created, and we wanted to maintain some "traceability" in terms of 301 redirects from already bookmarked pages, etc.